src/sentry/static/sentry/app/components/dugoutHelper.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Reflux from 'reflux';
+import classNames from 'classnames';
+import ApiMixin from '../mixins/apiMixin';
+import GuideStore from '../stores/guideStore';
+
+const DugoutHelper = React.createClass({
```

Comment:
Can you use the `create-react-class` package? React.createClass will be removed in React 16